src/components/InputTime/InputTime.tsx

```diff
@@ -31,138 +18,31 @@ interface InputTimeProps extends Omit<InputProps, 'value' | 'max' | 'min'> {
 }
 
 export const InputTime: React.FC<InputTimeProps> = ({
-  className = '',
-  forwardedRef = undefined,
   max,
   min,
-  onChange,
   value,
   ...passedProps
 }) => {
-  const mainRef = useRef<HTMLInputElement | null>(null);
-  const shadowTimeInputRef = useRef<HTMLInputElement | null>(null);
-
-  const syncValidity = () => {
```

Comment:
90% of `InputTime` is now in `InputTime/AsString`

`InputTime` is now a wrapper than conditionally renders either `AsString` or `AsDate`